src/bindings/js/node/include/infer_request.hpp

```diff
@@ -46,9 +46,20 @@ class InferRequestWrap : public Napi::ObjectWrap<InferRequestWrap> {
      * @param info[0] Javascript Tensor object.
      */
     Napi::Value set_input_tensor(const Napi::CallbackInfo& info);
-    /// @brief  Infers specified input in synchronous mode.
+
+    /** @brief  Infers specified inputs in synchronous mode.
+     * Checks incoming Napi::Value and calls the correct overload
+     */
     Napi::Value infer(const Napi::CallbackInfo& info);
 
+    /// @brief Infers specified inputs in synchronous mode.
+    /// @param inputs  An object with a collection of pairs key (input_name) and a value (tensor, tensor's data)
```

Comment:
```suggestion
    /**
     * @brief Infers specified inputs in synchronous mode.
     * @param inputs  An object with a collection of pairs key (input_name) and a value (tensor, tensor's data)
     */
```
Detail: Try use same comment stile to b consistent use `///` or `/*`.